import_map.json

```diff
@@ -19,7 +19,13 @@
     "@kubernetes/client-node": "npm:@kubernetes/client-node@0.18.1",
     "aws-sdk": "npm:aws-sdk@2.1229.0",
     "listr": "npm:listr@0.14.3",
-    "rxjs": "npm:rxjs@7.5.6"
+    "rxjs": "npm:rxjs@7.5.6",
+    "winston": "npm:winston@3.8.2",
+    "dots-wrapper": "npm:dots-wrapper@3.10.0",
```

Comment:
I think Michael removed our use of this library (not sure if it's used in your changes but may be able to be removed)